src/package/nuspec/Microsoft.TestPlatform.nuspec

```diff
@@ -0,0 +1,390 @@
+    <file src="net451\$Runtime$\QTAgent32_35.exe.Config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent32_35.exe.Config" />
+    <file src="net451\$Runtime$\QTAgent32_40.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent32_40.exe" />
+    <file src="net451\$Runtime$\QTAgent32_40.exe.Config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent32_40.exe.Config" />
+    <file src="net451\$Runtime$\QTAgent_35.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent_35.exe" />
+    <file src="net451\$Runtime$\QTAgent_35.exe.Config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent_35.exe.Config" />
+    <file src="net451\$Runtime$\QTAgent_40.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent_40.exe" />
+    <file src="net451\$Runtime$\QTAgent_40.exe.Config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTAgent_40.exe.Config" />
+    <file src="net451\$Runtime$\QTDCAgent.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTDCAgent.exe" />
+    <file src="net451\$Runtime$\QTDCAgent.exe.Config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTDCAgent.exe.Config" />
+    <file src="net451\$Runtime$\QTDCAgent32.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTDCAgent32.exe" />
+    <file src="net451\$Runtime$\QTDCAgent32.exe.Config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\QTDCAgent32.exe.Config" />
+    <file src="net451\$Runtime$\SummaryResult.xsd"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\SummaryResult.xsd" />
+    <file src="net451\$Runtime$\System.Collections.Immutable.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\System.Collections.Immutable.dll" />
+    <file src="net451\$Runtime$\System.Reflection.Metadata.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\System.Reflection.Metadata.dll" />
+    <file src="net451\$Runtime$\System.Runtime.InteropServices.RuntimeInformation.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\System.Runtime.InteropServices.RuntimeInformation.dll" />
+    <file src="net451\$Runtime$\testhost.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\testhost.exe" />
+    <file src="net451\$Runtime$\testhost.exe.config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\testhost.exe.config" />
+    <file src="net451\$Runtime$\testhost.x86.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\testhost.x86.exe" />
+    <file src="net451\$Runtime$\testhost.x86.exe.config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\testhost.x86.exe.config" />
+    <file src="net451\$Runtime$\ThirdPartyNotices.txt"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\ThirdPartyNotices.txt" />
+    <file src="net451\$Runtime$\vstest.console.exe"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\vstest.console.exe" />
```

Comment:
This will break people who already using `Microsoft.TestPlatform` package. Example Cloudbuild package build definition. They should use `Microsoft.TestPlatform.Portable` now onward.

Reply:
Logged the PathTooLongException issue here : 
https://github.com/Microsoft/vstest/issues/1214